privacyIDEA-impl/src/main/java/org/privacyidea/context/PIContext.java

```diff
@@ -17,12 +17,15 @@ public class PIContext extends BaseContext
     private String transactionID = null;
     @Nonnull
     private final String otpFieldHint;
+    @Nullable
+    private final Integer otpLength;

```

Comment:
i guess that should be `int` not `Integer`

Reply:
Integer is right. is returned from external function which changes String to Integer